c/libuv/libuv.go

```diff
@@ -297,7 +297,7 @@ func (handle *Handle) Fileno(fd *OsFd) c.Int {
 }
 
 //go:linkname UvPipe C.uv_pipe
-func UvPipe(fds [2]UvFile, readFlags c.Int, writeFlags c.Int) c.Int {
+func UvPipe(fds [2]File, readFlags c.Int, writeFlags c.Int) c.Int {
```

Comment:
```suggestion
func Pipe(fds [2]File, readFlags c.Int, writeFlags c.Int) c.Int {
```

Reply:
There is a repeated Struct named `pipe`, so we rename the func into `UvPipe`
- `libuv.go`
```go
type Pipe struct {
	Unused [0]byte
}
```

Reply:
maybe: type `uv_pipe_t` => `PipeT`, func  `uv_pipe` => `Pipe`.

Reply:
> maybe: type `uv_pipe_t` => `PipeT`, func `uv_pipe` => `Pipe`.

solved due to srtuct `Pipe`removed.